html/About.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+
+    <title> About Tobi Dele-Safa </title>
+    <link href="https://fonts.googleapis.com/css?family=Amatic+SC|Josefin+Slab|Montserrat|Raleway&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./css/main.css">
+    <link rel="stylesheet" href="./css/responsive.css">
+
+</head>
+
+<body>
+    <div class="header">
+        <!-- a logo is at the top of the navigation bar on the left hand side of the page -->
+        <a class="logo" href="">
+            <img src="https://via.placeholder.com/20x20" alt="logo">
```

Comment:
Most of the image elements have an `alt="logo"`. Please add more description to them. Eg: `alt="facebook logo"`